docs/source/standard.md

```diff
@@ -39,67 +39,92 @@ What the TRE organisation does to ensure information risk is measured and manage
+#### Project onboarding
 
-The ability of the TRE organisation to ensure that people with access to data are identified correctly and they are suitably qualified.
+| Statement | Guidance | Mandatory status |
+| --------- | -------- | ---------------- |
+| You have checks in place to ensure a project has the legal, financial and ethical requirements in place for the duration of the project | This includes checks that contracts are in place where required, adequate funding is available for the duration of the project, and responsibilities concerning data ownership are understood by all parties | Mandatory |
 
-#### 1.7.1 Training management
+#### Project closure
 
-The ability of the TRE organisation to track and maintain adequate training levels.
+| Statement | Guidance | Mandatory status |
+| --------- | -------- | ---------------- |
+| You have standard processes in place for the end of a project, that follow all legal requirements and data security best practice | This includes the archiving of quality and log data along with the archiving or deletion of data sets | Mandatory |
 
-| Statement | Guidance |
-| --------- | -------- |
-|           |          |
+#### Roles and responsibilities
 
-#### 1.7.2 Training delivery
+| Statement | Guidance | Mandatory status |
+| --------- | -------- | ---------------- |
+| You have clearly defined roles and responsibilities within your TRE organisation for all members | This may include roles such as users, system administrators, system operators, data providers and more. Every member of your TRE organisation should have a pre-defined role with clear powers and responsibilities | Mandatory |
 
-The ability of the TRE organisation to deliver training.
+### 1.7 Researcher accreditation
 
-| Statement | Guidance |
-| --------- | -------- |
-|           |          |
+The ability of the TRE organisation to ensure that people with access to data are identified correctly and they are suitably qualified.
+
+#### Onboarding members
+
+| Statement | Guidance | Mandatory status |
+| --------- | -------- | ---------------- |
+| You have clear onboarding processes in place for all roles within your TRE organisation | This may include all members signing role-specific terms of use, and completing role specific training | Mandatory
+| You have a robust method for identifying accredited members of your TRE organisation, prior to their accessing of sensitive data | This may include 2 factor authentication, ID checks or email verification | Mandatory |
+
+#### Training management and delivery
+
+| Statement | Guidance | Mandatory status |
+| --------- | -------- | ---------------- |
+| You have relevant training for all roles within the TRE organisation, and the ability to deliver this training | This may include: Cyber security training, GDPR training, and higher level training for system operators | Mandatory
+| All TRE organisation members have completed relevant training within the last 12 months | | Mandatory |
```

Comment:
How strict is `12 months`? Technically it means you have to complete the training on the same day or earlier in the year each time. Could say 13 months instead?

Reply:
12 months is a strict requirement for DSPT. I'm not sure it needs to be for us. How about "All TRE organisation members have recently completed relevant training, for example within the last twelve months."

Reply:
I reckon talking point for today, making it a clear timeframe helps with credibility